Clarify setting of library environment variables in module examples (#3730)
godotengine/godot-docs: development/cpp/custom_modules_in_cpp.rst

    # First, create a custom env for the shared library.
    module_env = env.Clone()

>     # Position independent code is required for a shared library.

Reviewer: ```suggestion
    # Position-independent code is required for a shared library.
```

    module_env.Append(CCFLAGS=['-fPIC'])

>     # Don't inject godot's dependencies into our shared library.

Reviewer: ```suggestion
    # Don't inject Godot's dependencies into our shared library.
```

    module_env['LIBS'] = []

>     # Define the shared library. By default it would be built in the module's

Reviewer: ```suggestion
    # Define the shared library. By default, it would be built in the module's
```

    # folder, however it's better to output it into `bin` next to the
    # Godot binary.
    shared_lib = module_env.SharedLibrary(target='#bin/summator', source=sources)

    # Finally, notify the main build environment it now has our shared library
    # as a new dependency.

>     # LIBPATH and LIBS need to be set on the real "env" (not the clone) in order
>     # to link the specified libs to the godot executable.

Reviewer: ```suggestion
    # LIBPATH and LIBS need to be set on the real "env" (not the clone)
    # to link the specified libraries to the Godot executable.
```
